jsdoc/tag-defs/access.js

```diff
@@ -1,13 +1,16 @@
-module.exports = function(accessTagTransform, trimWhitespaceTransform) {
-  var name = 'access'
-  accessTagTransform.addTag(name);
-  accessTagTransform.addValue('private');
-  accessTagTransform.addValue('protected');
-  accessTagTransform.addValue('public');
-
+module.exports = function(createDocMessage) {
+  STANDARD_VALUES = ['private', 'protected', 'public'];
   return {
-    name: name,
-    docProperty: 'access',
-    transforms: [trimWhitespaceTransform, accessTagTransform]
+    name: 'access',
+    transforms: function(doc, tag, value) {
+      var value = value.trim();
+      if (STANDARD_VALUES.indexOf(value) !== -1) {
+        if (doc.tags.getTags(value).length > 0) {
+          throw new Error(createDocMessage('"@access ' + value + '" tag cannot be used with "@' + value + '" tag on the same document', doc));
```

Comment:
What I was trying to indicate was that my document may contain `@access private` twice.

The message that I can't use `@access private` with `@private` may be a little confusing if the error is actually caused by having two `@access private` lines.